.gitignore

```diff
@@ -1,3 +1,2 @@
 .idea
-composer.lock
```

Comment:
Lest I forget, thank you for providing your insights. I appreciate those as I always learn from such an exchange